src/geocat/viz/util.py

```diff
@@ -861,6 +863,62 @@ def findLocalExtrema(da: xr.DataArray,
                      lowVal: int = 1000,
                      eType: str = 'Low',
                      eps: float = 10) -> list:
+    """This function is deprecated. Call `find_local_extrema` instead.
```

Comment:
If you're interested, this is actually how we handle the docstrings of deprecated functions in comp

https://github.com/NCAR/geocat-comp/blob/bf184d4ca8ffd91061ab066dc0a79bccaf3fc19d/src/geocat/comp/climatologies.py#L139-L143

This adds a custom admonition that's super visible in the documentation. We also move the deprecated function to a different header in the index file.

Reply:
Thanks done